### ISSUE TITLE
Game events should be updated against the station

### ISSUE THREAD

**Issue by metalgearsloth** [space-wizards/space-station-14#33823]
Right now events get added to the station but never actually have the station passed through, so events realistically will run against a random station that round.

**Comment by deltanedas** [comment]
related, stationdata needs separating actual station grids from owned grids like the cargo shuttle. this would let events apply effects like foam and spawns only to the station and not random shittles

**Comment by IProduceWidgets** [comment]
> related, stationdata needs separating actual station grids from owned grids like the cargo shuttle. this would let events apply effects like foam and spawns only to the station and not random shittles

Yeah I do think this is the right way to go, I just haven't gotten back to poking the ATS mob spawning pr thing.
I don't have a good timeline on when I will get back to that though.
